### ISSUE TITLE
Sign out Confirmation - Accessibility issues

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#24367]
## What happened?
(Sign out confirmations on profile page and settings page.)

1. "Are you sure you want to sign out?" should be labeled a header (header 2) (Android and iOS)
2. AccessibilityHint on Confirm button and Cancel button has "VEE A" pronounced as "vah" on Samsung mobiles.

## Specs:
- Device: Samsung phone

## Steps to Reproduce
Enable TalkBack or VoiceOver

1. Read "Are you sure you want to sign out?" with TalkBack or VoiceOver.

2. Read buttons with Samsung phones.

## Desired behavior
1. Text is marked as header.

2. Samsung phones say "Vee A" instead of "vah"

## Acceptance Criteria
See above.

## Bug Severity
- 4 - Low

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/23498

## Screen shot(s) and additional information

![image.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/e7df8620-112e-429a-869a-10760dd5c1e4)

**Comment by digitalken** [comment]
PASS 5/6/2021 - Looks good on iOS and Android.  Minor bug for lack of header being marked in accessibility and Samsun pronunciation of Vee A. (vah).  Closing.

**Comment by digitalken** [comment]
PASS 6/17/2021 - Sounds good on iOS and Android.  Closing.
